lib/iris/coords.py

```diff
@@ -671,6 +679,32 @@ def _bounds_setter(self, bounds):
 
     bounds = property(_bounds_getter, _bounds_setter)
 
+    def _climatology_getter(self):
```

Comment:
@trexfeathers It's like your subconscious is trying to tell you that `climatology` is the proper name of the attribute and not `bounds_are_climatological` :stuck_out_tongue_winking_eye:

Reply:
Stop, my mind is feeling more dualistic by the minute 🤢